test/v1/support/IfsResponseMappingSupportSpec.scala

```diff
@@ -47,14 +48,14 @@ class IfsResponseMappingSupportSpec extends UnitSpec {
     "single error" when {
       "the error code is in the map provided" must {
         "use the mapping and wrap" in {
-          mapping.mapIfsErrors(errorCodeMap)(ResponseWrapper(correlationId, IfsErrors.single(IfsErrorCode("ERR1")))) shouldBe
+          mapping.mapIfsErrors(errorCodeMap)(ResponseWrapper(correlationId, DownstreamErrors.single(DownstreamErrorCode("ERR1")))) shouldBe
             ErrorWrapper(correlationId, Error1)
```

Comment:
Again, maybe change file name to `DownstreamResponseMappingSupportSpec` and if so rename references to IFS to Downstream?